docs/source/en/model_doc/speech_to_text.mdx

```diff
@@ -47,23 +47,15 @@ be installed as follows: `apt install libsndfile1-dev`
 >>> import torch
 >>> from transformers import Speech2TextProcessor, Speech2TextForConditionalGeneration
 >>> from datasets import load_dataset
->>> import soundfile as sf
 
 >>> model = Speech2TextForConditionalGeneration.from_pretrained("facebook/s2t-small-librispeech-asr")
 >>> processor = Speech2TextProcessor.from_pretrained("facebook/s2t-small-librispeech-asr")
 
 
->>> def map_to_array(batch):
-...     speech, _ = sf.read(batch["file"])
-...     batch["speech"] = speech
-...     return batch
-
-
 >>> ds = load_dataset("hf-internal-testing/librispeech_asr_dummy", "clean", split="validation")
```

Comment:
I saw we use `hf-internal-testing/librispeech_asr_demo` in `PT_SPEECH_BASE_MODEL_SAMPLE` and in other PT speech models in `doc.py`. Not sure if there is any difference between these 2 datasets, but maybe use `librispeech_asr_demo` in all places is good (if this makes sense)